### PR TITLE
feat: added header, sidebar and navigation footer

### PR DESCRIPTION
**Pull request by JaanviNL**

### CODE REVIEW

File: components/badge/badge.html [hunk 1]
```diff
@@ -36,50 +37,53 @@ <h4>Components</h4>
...
-    <hr class="break-line">
-    <h3>Avatar with Badge</h3>
-    <p class="content-text">Avatar with badge are used to display the user's status which can be online , offline and do-not-disturb</p>
+            <iframe src="https://carbon.now.sh/embed/sHFaUtFpR68svbwiqkQF"
+                style="width: 684px; height: 426px; border:0; transform: scale(1); overflow:hidden;" class="code-img"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+
+
+            <hr class="break-line">
```
Comment (rohanmathur91) [inline components/badge/badge.html:76]: Remove unnecessary space

File: components/navigation/nav.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
+@import url(/main.css);
+@import url(/style.css);
+
+
+
+.sidebar-nav
```
Comment (rohanmathur91) [inline components/navigation/nav.css:6]: same as above

File: components/navigation/nav.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+{
+    display: flex;
+    flex-direction: column;
+    width: 15rem;
+    padding: 2rem;
+}
+h2{
+    margin-top: 1rem;
+}
```
Comment (rohanmathur91) [inline components/navigation/nav.css:15]: use any tool/extension for proper spacing.

File: components/navigation/nav.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+.header-b
+{
+    color: white;
+    display: flex;
+    justify-content: stretch;
+    
+}
+.content-nav
+{
```
Comment (rohanmathur91) [inline components/navigation/nav.css:54]: have a proper spacing between classes

**Review by hiteshverma27 — APPROVED** [review]
Use a code formatter like prettier to format code

**Review by rohanmathur91 — APPROVED** [review]
Have a proper indentation and spacing, rest looks good.

**Pull request merged**